priceloop_api/priceloop/extras/table.py

```diff
@@ -0,0 +1,855 @@
+import json
+from enum import Enum
+from http import HTTPStatus
+from io import StringIO
+from typing import Any, Callable, List, Literal, Optional
+
+import attr
+import numpy as np
+import pandas as pd
+import requests
+from pandas.api.types import infer_dtype
+from retry import retry
+from retry.api import retry_call
+
+from priceloop_api.api.import_api import create_csv_import_job, get_csv_import_job
+from priceloop_api.api.table_api import (
+    create_table,
+    delete_table,
+    get_incomplete_table_query_data,
+    get_table,
+    get_table_data,
+    patch_table_data,
+    truncate_table,
+)
+from priceloop_api.client import AuthenticatedClient
+from priceloop_api.errors import UnexpectedStatus
+from priceloop_api.models import (
+    ApiColumn,
+    ApiColumnSchema,
+    ApiColumnWithNullableValues,
+    ApiColumnWithValues,
+    ApiOnConflict,
+    ApiTable,
+    ApiTableData,
+    ApiTableSchema,
+    Append,
+    Delete,
+    ExplicitExprType,
+    ImportJob,
+    Insert,
+    TableImportMode,
+    TableRow,
+    Update,
+)
+from priceloop_api.types import UNSET, Unset
+
+from .exceptions import (
+    ColumnMixedTypes,
+    ColumnsMismatch,
+    ColumnTypesMismatch,
+    PlatformError,
+    )
+
+LIMIT = 30000  # cells
+
+
+COLUMNS_TYPES_MAPPING = {
```

Comment:
should we name it consistent with line 65 so something like `PANDAS_TO_PLATFORM_MAPPING`?